src/test/java/rs117/hd/lighting/LightConfigParser.java

```diff
@@ -8,6 +8,9 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import lombok.extern.slf4j.Slf4j;
+import rs117.hd.scene.lighting.Alignment;
+import rs117.hd.scene.lighting.Light;
+import rs117.hd.scene.lighting.LightType;
 
 @Slf4j
 public class LightConfigParser
```

Comment:
The parser still references the deleted `lights.txt`

Reply:
https://github.com/RS117/RLHD/blob/d51edeeed7bd29bdcdb6e9a97466947dd0044a6c/src/test/java/rs117/hd/lighting/LightConfigParser.java#L45